deploy/01_genesis.ts

```diff
@@ -33,22 +33,25 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
   }
   if (
     holographGenesisContract != null &&
-    holographGenesisContract.address == '0x9f69aefbb4418a1b4642116b5c8c2b30896019a8'
+    holographGenesisContract.address == '0x0C8aF56F7650a6E3685188d212044338c21d3F73'
```

Comment:
this has already been deployed at this address

Reply:
This is for localhost deployments